test/setup/make_source.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "#!/usr/bin/env bash" > ../source.sh
+
+project_id=$(terraform output project_id)
+echo "export TF_VAR_project_id='$project_id'" >> ../source.sh
+echo "export TF_VAR_pubsub_project_id='$project_id'" >> ../source.sh
+
+network_project=$(terraform output network_project)
+echo "export TF_VAR_network_project='$network_project'" >> ../source.sh
+
+enforcer_project_id=$(terraform output enforcer_project_id)
+echo "export TF_VAR_enforcer_project_id='$enforcer_project_id'" >> ../source.sh
+echo "export TF_VAR_sink_project_id='$enforcer_project_id'" >> ../source.sh
```

Comment:
several echo to same file was causing lint errors for me on other project, did linter ran well here?

Reply:
Yes